lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -4412,6 +4412,30 @@ would appear as ``$options.vcf_output`` having the values ``'--vcf'`` when true
       </data>
     </outputs>
 ```
+
+Variables that have the type attribute ``select`` need a special python statement of
+the form ``selection and "option" in selection`` inside the ``<filter>`` tags. 
+
+### Example
+
+```xml
+    <inputs>
+       <param name="outputs" type="select" optional="true">
+            <option value="save_phase">Phase Movie</option>
+            <option value="save_period">Period Movie</option>
+        </param>
+    </inputs>
+
+    <outputs>
+        <data name="phase_out" format="tiff" label="${movie.name[:-4]}_phase">
+            <filter>outputs and "save_phase" in outputs</filter>
+        </data>
```

Comment:
```suggestion
        <data name="phase_out" format="tiff">
            <filter>output_type and "save_phase" in output_type</filter>
        </data>
```

would suggest not to advocate such a non standard label

Reply:
ok, for the exmaple I understand. For my tool getting rid of the `.tif` improved the outputs filenames quite a lot, otherwise you get input.tif.tiff and such things..